### PR TITLE
Admin - Add parameters to choose Redis backend host and port

### PR DESCRIPTION
**Pull request by mdouchin**
@rldhont  locales have been created for english and french in this PR, please give guidelines to add other locales

### CODE REVIEW

**Comment by laurentj** [comment]
Parameters in the form should be hidden if the storage type is not redis, IMHO

File: lizmap/modules/lizmap/classes/lizmapProxy.class.php [hunk 1]
```diff
@@ -392,11 +392,18 @@ static public function createVirtualProfile( $repository, $project, $layers, $cr
...
+                $cacheRedisPort = trim($ser->cacheRedisPort);
+
             // Virtual cache profile parameter
             $cacheParams = array(
                 "driver"=>"redis",
-                "host"=>"localhost",
-                "port"=>"6379",
+                "host"=>$cacheRedisHost,
+                "port"=>$cacheRedisPort,
                 "ttl"=>$cacheExpiration
```
Comment (laurentj) [inline lizmap/modules/lizmap/classes/lizmapProxy.class.php]: a "db" parameter should be added (optional, integer >=0), in order to allow to select the database index in Redis.

Comment (laurentj) [inline lizmap/modules/lizmap/classes/lizmapProxy.class.php]: a "key_prefix" parameter should be added (optional, string), in order to indicate key_prefix.

**Comment by mdouchin** [comment]
@laurentj  @rldhont updated with key prefix and db optional parameters + hide them in form if redis is not chosen

**Comment by laurentj** [comment]
It ok for me :-)

**Pull request merged**